lex.go

```diff
@@ -189,7 +190,8 @@ func lexVariable(l *lexer) stateFn {
 				}
 				return lexValue
 			}
-			return l.errorf("invalid variable: %s", variable)
+			fmt.Fprintf(os.Stderr, "ignoring variable: %s\n", variable)
+			return lexValue
```

Comment:
I'm not a fan of printing to std{out,err} from inside the library, I would rather just ignore the variable silently. Well I would prefer it was implemented such that it can handle this correctly, but I'm ok with ignoring it for now :)